raiden-cli/raspberrypi.md

```diff
@@ -0,0 +1,92 @@
+# Running raiden-cli on the raspberry pi
+
+This tutorial assumes raspberry pi to be in the same LAN as your PC with SSH enabled
+These steps have to be run on the raspberry pi 4. The steps to copy files from your `PC` to the `pi` have been explicitly mentioned and have to be run on your `PC`.
+
+`sudo apt update`
```

Comment:
Rather put those lists of commands in a code block
```
```sh
```

Reply:
done